RollGen/Utils.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace RollGen
+{
+    public static class Utils
```

Comment:
@cidthecoatrack The reason not to is because it needs to be accessible by other projects.